.travis.yml

```diff
@@ -33,24 +33,10 @@ jobs:
         skip_cleanup: true
         on:
           tags: true
-  - name: "Build & Deploy Helm"
-    script:
-      - docker run -v $TRAVIS_BUILD_DIR/deployment/helm/:/helm  -w /helm --entrypoint /helm/build.sh alpine/helm:3.2.1 $TRAVIS_TAG
-    deploy:
-      - provider: releases
-        draft: true
-        skip_cleanup: true
-        overwrite: true
-        on:
-          tags: true
-        api_key:
-          secure: dPDprfJo3c5iCk5fcPR7tgRMz5OcCF6IAPRx1/WMAq1FJjgdkD6iu628/AAgezN/rlTDiqZpfx3nj3UhTSUOeBZgEaRbpvpIc3bLc2RABqqxQNuL2g+DNt3L7XYcq8PLHQBVIHmRKp8DK1XJCaG3LUEuYZD9gKaiftMwbvgsgfVOFasslyypgDTi/NdPm1tNsDdKvYVe9dkZpDpHClJsoIQLl/gtNvrJREFTF2w0T8oSdccsoQekAAC4kXWiL5lgXIKHDpQTo1K8oXrl/hUxuw0e1fGKA6R0kNZliA6cQYG3OOb0a9JhplzO3jGWvlZdmJluWgy0yUn+jRZZOXiQVJOZCd2Lht3JN8p6x9D91S43forMV7ZmrW3XPuEocGIwVdY7yVKEsHAiAK3Ih3VbMuUn1ITvKB8F4idyurWEL9cQwsszX9pFSOriAH5mhIQs7vwp6uvFJIoADGiS//wEqcAwzsFWHCDLBPYRTk9Zp17MSgUj8+PDBo29pbbWtx6+14w82LVQQdnyoReDiZR5G1ItxSx6xjWIeGlyo8yu4xmNq9nufPoj6ZkGDbAJtH2YeaGIEcltjXzIalmBdmoVp+6SKyr8EFnolgWd+ns8KkCvLqxp3fwuz4OD+B619Au1/8WUrHRdetChlvxzEHVx3wnwWnw0zbCBh/cq0mWUdzs=
-        file_glob: true
```

Comment:
I suspect we still need this to allow `postfacto-*.tgz`

Reply:
I've added the property to the deploy action of "Build & Test Package". Is there a way to test it works properly with a draft release?

Reply:
I don't think there's a way to test it really. We know it worked from the "Build & Deploy Helm" step, and it matches the description in https://docs.travis-ci.com/user/deployment/releases/#uploading-multiple-files

Reply:
We have tested it by pushing to my fork of Postfacto, and that managed to upload https://github.com/j-lea/postfacto/releases/tag/j8

Reply:
I am going to resolve this thread, reopen if you need more info on this.